providers/aws/aws_provider.go

```diff
@@ -33,6 +33,16 @@ type AWSProvider struct {
 
 const awsProviderVersion = ">2.25.0"
 
+// global resources should be bound to a default region. AWS doesn't specify in which region default services are
+// placed (see  https://docs.aws.amazon.com/general/latest/gr/rande.html), so we shouldn't assume any region as well
+var SupportedGlobalResources = []string{
```

Comment:
good idea move this to aws provider

Reply:
At first I was thinking to define a method on every AWS resource to enforce checking whether a given resource is global or not but this list now looks to be easy to spot and hope to do the job.